medical_medicament/models/medical_medicament.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# #############################################################################
+#
+# Tech-Receptives Solutions Pvt. Ltd.
+# Copyright (C) 2004-TODAY Tech-Receptives(<http://www.techreceptives.com>)
+# Special Credit and Thanks to Thymbra Latinoamericana S.A.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# #############################################################################
+
+from openerp import models, fields, api
+
+
+class MedicalMedicament(models.Model):
+    _name = 'medical.medicament'
+    _inherit = ['mail.thread']
+    _inherits = {'product.product': 'product_id'}
+
+    @api.multi
+    def onchange_type(self, _type):
+        return self.product_id.onchange_type(_type)
+
+    @api.multi
+    def onchange_uom(self, uom_id, uom_po_id):
+        return self.product_id.onchange_uom(uom_id, uom_po_id)
+
+    @api.multi
+    def name_get(self):
+        res = []
+        for rec in self:
+            name = '%s - %s' % (self.product_id.name, self.drug_form_id.name)
```

Comment:
@JonathanNEMRY - Ran into an issue, this should be `name = '%s - %s' % (rec.product_id.name, rec.drug_form_id.name)`